### PR TITLE
Setup notification service in desktop apps

### PR DESCRIPTION
**Pull request by adube**
This PR introduces the notification service in the mobile app.

## Todo

 * [x] review

## Live demo

Try to login with bad credentials.  The geolocation tool isn't there in those pages (is this a bug ?)

 * https://adube.github.io/ngeo/notification-in-desktop-apps/examples/contribs/gmf/apps/desktop/
 * https://adube.github.io/ngeo/notification-in-desktop-apps/examples/contribs/gmf/apps/desktop_alt/

### CODE REVIEW

File: contribs/gmf/less/desktop.less [hunk 1]
```diff
@@ -400,4 +400,16 @@ gmf-featurestyle {
...
+/**
+ * Notifications
+ */
+.ngeo-notification {
+  left: 50%;
+  margin: 0 0 0 -100px;
+  position: absolute;
+  top: 0;
+  width: 200px;
 }
```
Comment (sbrunner) [inline contribs/gmf/less/desktop.less]: Missing end line...

**Comment by sbrunner** [comment]
Just a relay small comment, also that's looks good :-)

**Comment by adube** [comment]
Fixed.

**Comment by sbrunner** [comment]
@fredj for me it can be merged :-)

File: contribs/gmf/less/desktop.less [hunk 2]
```diff
@@ -400,4 +400,16 @@ gmf-featurestyle {
...
\ No newline at end of file
+}
+
+
+/**
+ * Notifications
+ */
+.ngeo-notification {
+  left: 50%;
+  margin: 0 0 0 -100px;
```
Comment (sbrunner) [inline contribs/gmf/less/desktop.less]: we should use rem instance of px...

**Comment by sbrunner** [comment]
Finally just a remark about the unit, sorry @adube

**Comment by fredj** [comment]
@sbrunner please merge when your comment is addressed

**Comment by adube** [comment]
Fixed.  Live demo not updated.

**Pull request merged**